packages/tests/wdio.conf.js

```diff
@@ -1,3 +1,13 @@
+const Logger = require('./logger.js');
+
+const testPath = `.`;
```

Comment:
btw no need to use backticks ` when not interpolating values, a standard single quote will sufice